CMakeLists.txt

```diff
@@ -229,6 +229,27 @@ endif()
 # Compiler Settings #
 #####################
 
+# needed to check if a C compiler flag is possible
+include(CheckCCompilerFlag)
+function(check_cc_flag CC_FLAG)
```

Comment:
Can you put these functions into a different file, e.g. somewhere here:
https://github.com/open62541/open62541/tree/master/tools/cmake